migrations/20250315140840_add_merge_state_to_pr.up.sql

```diff
@@ -0,0 +1,2 @@
+-- Add up migration script here
+ALTER TABLE pull_request ADD COLUMN merge_state TEXT NOT NULL DEFAULT 'unknown';
```

Comment:
I would also call it `mergeable_state`, because `merge_state` sounds like it was already merged.